.github/workflows/workflow.yaml

```diff
@@ -317,6 +317,59 @@ jobs:
             done
           done
 
+  benchmark:
+    name: "Benchmark"
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v3
+
+      - name: Download release benchmark file
+        uses: robinraju/release-downloader@v1.6
+        id: get-latest-benchmark
+        continue-on-error: true
+        with: 
+          latest: true
```

Comment:
is this going to take latest semver or latest release?

Reply:
Latest

Reply:
I couldn't find an action that returns the latest semver. Still looking, I think we could sort it also after retrieving every release if this is needed.

Reply:
can we use the github api to get recent releases then find the latest semver?

Reply:
Yip! It took some time, but I was able to get the latest release semver version by calling API and sorting